app/stylesheets/main.scss

```diff
@@ -1,6 +1,14 @@
+@import '~stylekit';
 @import '~highlightjs_css';
 
-body, html {
+:root {
+  --sn-stylekit-monospace-font: SFMono-Regular, Consolas, Liberation Mono, Menlo,
```

Comment:
I don't think it makes sense to override stylekit variables. The point of stylekit is to provide uniformity amongst editors. Even if you did think this was important, this PR should only address syntactical changes, and not functionality or appearance. This should be left to a separate PR.

Reply:
The stylekit only has ` 'Ubuntu Mono', courier, monospace;` for monospace font (shown in next line). I got the others directly from GitHub. Are you sure you don't want to update the fonts?

Reply:
It's not that we don't want to. It's just that it should be addressed in a separate PR.

Reply:
OK